pkgs/by-name/li/live555/package.nix

```diff
@@ -78,8 +78,15 @@ stdenv.mkDerivation (finalAttrs: {
 
   makeFlags = [
     "PREFIX=${placeholder "out"}"
+    "C_COMPILER=$(CC)"
+    "CPLUSPLUS_COMPILER=$(CXX)"
+    "LIBRARY_LINK=$(AR) cr "
+    "LINK=$(CXX) -o "
   ];
 
+  # required for whitespaces in makeFlags
+  __structuredAttrs = true;
```

Comment:
Where is this documented?

Reply:
https://nixos.org/manual/nix/stable/language/advanced-attributes#adv-attr-structuredAttrs